xdn-nx-example/XDN.md

```diff
@@ -0,0 +1,203 @@
+# XDN
+
+## Development: run NX app
+
+```sh
+yarn start
+```
+
+## Build
+
+```sh
+yarn xdn:build
+```
+
+## Production: run XDN app
+
+```sh
+yarn xdn:run
+```
+
+## Deployment: XDN
+
+```sh
+yarn xdn:deploy
+```
```

Comment:
Should this be part of the README.md? Since this example already has Nx and XDN setup, the README.md should have clear instructions at the top on how to launch the project.

Reply:
I added this guide in my https://github.com/moovweb-docs/xdn-docs/pull/125 PR as well, also linked this guide in the main `README.md` file. Just decided to hold it on this project as well to be able to know the main commands for development